drive/drive_test/drive_test.go

```diff
@@ -0,0 +1,465 @@
+package drive_test
```

Comment:
You can have `x_test` packages in `x` folder, i.e. there is no need to create a new package called `driver_test`. Simply move this file into `drive` folder.